schema-guide.md

```diff
@@ -2698,16 +2698,26 @@ authors:
 
 ### `definitions.reference.end`
 
-- **type**: `integer`
+- **type**: Nonempty string or `integer`
```

Comment:
```suggestion
- **type**: Nonempty `string` or `integer`
```

Formatting.

Reply:
Done in https://github.com/citation-file-format/citation-file-format/pull/286/commits/4478519e720370bc87d4746240d53bf4fba27f6d